### PR TITLE
Add note about Fn+F4 for the 14" model and 16" model

### PR DESCRIPTION
**Pull request by ahoneybun**
The darp10-b has the backlight set for Fn+F4 but we only note about the darp10 which has it on Fn+* due to the Numpad.

### CODE REVIEW

File: src/models/darp10/external-overview.md
Comment (leviport) [inline src/models/darp10/external-overview.md:83]: This should probably also have an asterisk and a footnote, since the 16" does not have a toggle hotkey on F4

**Review by leviport — CHANGES_REQUESTED** [review]

File: src/models/darp10/external-overview.md
Comment (leviport) [inline src/models/darp10/external-overview.md:93]: Actually, the next three hotkeys probably all need the note for being on the 16" model only.

Also, you'll need a different marker for the 14" and 16" notes. Otherwise I can't tell which asterisk corresponds to which note. Two asterisks should suffice on one of the model.

**Review by leviport — CHANGES_REQUESTED** [review]

**Comment by ahoneybun** [comment]
I think that is the best I can get with Markdown.

File: src/models/darp10/external-overview.md
Comment (ahoneybun) [reply src/models/darp10/external-overview.md]: That looks like it works! I was a bit wary to add HTML in there but I guess it works.

**Review by leviport — APPROVED** [review]
This looks good to me, but let's make sure we get Jacob's blessing before we merge, just in case he prefers this be done a different way.

**Comment by jacobgkau** [comment]
This is what you had:

screenshot-2024-07-11-22-28-42
screenshot-2024-07-11-22-28-47

Not optimal that the top asterisk and the meaning at the bottom can't be in view at the same time when maximized on a normal 1080p display. I'm updating it to look like this instead:

screenshot-2024-07-11-22-32-21
screenshot-2024-07-11-22-32-26

No scrolling, and matches how it's phrased on the rest of the page better. We could get away with just "14:" and "16:", but I think we might as well do the whole thing since the icons are taking up the vertical space anyway.

**Comment by leviport** [comment]
I like that better as well!

**Review by leviport — APPROVED** [review]

**Review by jacobgkau — APPROVED** [review]

**Review by leviport — APPROVED** [review]

**Pull request merged**
